ios/FluentUI/PreviewCard/PreviewCard.swift

```diff
@@ -31,16 +33,21 @@ import SwiftUI
 /// View that represents the PreviewCard.
 public struct PreviewCard: View {
     var isElevated: Bool
+    var image: UIImage
     @Environment(\.theme) var theme: FluentUIStyle
     @Environment(\.windowProvider) var windowProvider: FluentUIWindowProvider?
     /// Creates inner PreviewCard view.
     @ViewBuilder
     var innerContents: some View {
         HStack {
             Spacer()
-                /// Sets the height and width of the Preview Card.
+                /// Creates placeholder image over PreviewCard.
```

Comment:
Nit: still says placeholder here. All three of these comments may not be fully necessary, either, but not a big deal.